syntax = "proto2";
package event;
message Events {
  repeated Event event = 1;
}

message Event {
 optional uint32 time = 1;
 required uint32 type_id = 2;
 required uint32 config_id = 3;
 required uint32 devid = 4;
 optional string obj = 5;
 optional uint32 thres_value = 6;
 optional uint32 alarm_value = 7;
 optional string value_type = 8;
}
